public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -924,6 +924,7 @@ export class PrometheusDatasource
             <img
               style={{ width: 14, height: 14, verticalAlign: 'text-bottom' }}
               src={LOGOS[buildInfo.application ?? PromApplication.Prometheus]}
+              alt="datasource subtype logo"
```

Comment:
this one also feels decorative

```suggestion
              alt=""
```